src/io/CavityWriter.cpp

```diff
@@ -10,96 +10,253 @@
 #include "particleContainer/ParticleContainer.h"
 #include "utils/Logger.h"
 #include "Simulation.h"
+#include "ensemble/CavityEnsemble.h"
 
 using Log::global_log;
 using namespace std;
 
-CavityWriter::CavityWriter(unsigned long writeFrequency, string outputPrefix, bool incremental) {
-	_outputPrefix= outputPrefix;
-	_writeFrequency = writeFrequency;
-	_incremental = incremental;
-
-	if (outputPrefix == "default") {
-		_appendTimestamp = true;
-	}
-	else {
-		_appendTimestamp = false;
-	}
-}
-
-CavityWriter::~CavityWriter(){}
-
+/** @brief reads in parameters for CavityEnsemble and output
```

Comment:
documentation should be in header, not in cpp file

Reply:
Changed as requested c9a26ad90439b66e658c293bae6cb914eba4b29c